thunder/core/utils.py

```diff
@@ -658,16 +662,27 @@ def _reify(x):
     check(len(dims) == len(set(dims)), lambda: f"Duplicate value in list of dimensions {dims}!")
 
 
+def is_cpu_scalar_tensor(a: Any) -> bool:
```

Comment:
@kiya00, would you add a doc string explaining this function?

nit: let's require `a` be a position argument

```
def is_cpu_scalar_tensor(a: Any, /) -> bool:
```